core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/V1WorkspaceMigrationMonitorSpec.scala

```diff
@@ -82,4 +84,37 @@ class V1WorkspaceMigrationMonitorSpec
     }
   }
 
+  "claimAndConfigureGoogleProject" should "return a valid database operation" in {
+    val spec = new WorkspaceServiceSpec()
```

Comment:
Instantiating a different test spec here doesn't feel quite right to me. Are we doing this because we need a `WorkspaceService` to test with?

Reply:
Agreed, but I need the mocked out workspace spec and I really didn't want refactor this hierarchy in this change.

Reply:
I tried to refactor this and died. It's a lot of work for very little benefit.

Reply:
Oof, I thought `TestApiService` and its related fixtures were in `TestDriverComponent`, not `WorkspaceServiceSpec`. Still seems like we must get around the need for a mocked workspace service in other tests, but I wouldn't be surprised at all if we didn't and this refactor isn't worth dying over